### DISCUSSION TITLE
[BUG] aws(elastickbeanstalk) and library

### DISCUSSION THREAD

**Discussion by AntonButyrin** [NilCoalescing/djangochannelsrestframework#191] [Q&A]
When i'm up code on localmachine @model_observer work, but if I'm deploy it on aws (elastickbeanstalk) @action work, but @model_observer don't work.I asked amazon support what the problem might be, we checked the server settings/code settings for the deployment and they indicated that most likely the problem is in the library.
I can subscribe/unsubscribe, but changes on model i can't get

class MethodConsumer(AsyncAPIConsumer)
![image](https://user-images.githubusercontent.com/66861091/214901581-265befb6-e073-4ee4-9d91-2421055e6f1f.png)

**Comment by hishnash** [comment]
Sorry I somehow missed this message in my inbox.

Do yo know what channel layer you are using?
